extensions/amp-consent/0.1/cmps.js

```diff
@@ -76,3 +76,8 @@ CMP_CONFIG['Usercentrics'] = {
   'checkConsentHref': 'https://consents.usercentrics.eu/amp/checkConsent',
   'promptUISrc': 'https://amp.usercentrics.eu/amp.html',
 };
+
+CMP_CONFIG['iubenda'] = {
```

Comment:
A `promptUiSrc` must be defined that returns an iframe with the consent dialog.

Reply:
@micajuine-ho As I mentioned in the PR description, we can't define a promptUISrc since we are asking our customers to host it in their own website. They will set it manually with their own URI.

Reply:
> > * promptUI has not been specified in the configuration cause it's different for each customer. We ask our customers to host their promptUI on their own website, so this property must be set by customers.
> 
> Hmmm, so to clarify, the service you're offering is not the UI but the storage and retrieval of user-consent, which will then trigger amp-consent to either show or not show the consent dialog provided by the publisher?

Exactly.

> 
> This functionality is not currently provided by amp-consent as we require the CMPs provide both the UI (`promptUISrc`) as well as the consent management (`checkConsentHref`). Is this a required feature for you?
> 

As you can see in cmps.js are currently using `checkConsentHref`, we ask our customers to manually set the promptUI so this is fine right?

> > * There was no way to test the added configuration locally (cmps.js) since there are no instructions on how to build and use AMP locally, in addition the example page is using the AMP version in production so I guess that I must wait the configuration to be live before testing.
> 
> Please see this guide to testing locally: https://github.com/ampproject/amphtml/blob/master/contributing/TESTING.md.

Thanks!

> 
> > * We would like to load the consent iframe even after the consent has been given (promptUI), at each pageview, cause we count the pageviews of our product. At this time there is only one way to load the iframe at each page view: by returning expireCache: true at each request to our check consent endpoint. But this clears the consent string. What do you think about creating another option like forcePromptUI to load the prompt UI at each page view? Thanks.
> 
> Does [postPromptUI](https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#post-prompt-ui-optional) work in this case? When defined, it should always show, and you can make it an amp-iframe.

Thank you, I will try. Is there a way to retrieve the current consentString from within the postPromptUI amp-iframe?